fastlane/Fastfile

```diff
@@ -574,8 +575,9 @@ REPOSITORY_NAME = 'WordPress-Android'
           aab: aab_file_path,
           track: options[:track],
           release_status: 'draft',
-          skip_upload_metadata: true,
-          skip_upload_changelogs: true,
+          metadata_path: metadata_dir,
+          skip_upload_metadata: (options[:track] != 'production'), # Only update app title/description/etc. if uploading for Production, skip for alpha/beta tracks
+          skip_upload_changelogs: false,
```

Comment:
This change is part of my plan for this sprint to automate:
 - The upload of PlayStore metadata (app title, short and long description, etc) by the CI every time we ship a new production release
    - as opposed to only do it manually on demand and when we have a request to update them, which was how it was done so far…
    - … which led to the metadata in the PlayStore not always being in sync with the `.txt` files we had in the repo, not allowing us to use the git history to know about which changes were done in the past and when
    - while now we'll guarantee to have those in sync for every production release
 - The upload of `changelogs/*.txt` file, which was previously done manually by copy/pasting the `fastlane/{jetpack_metadata,metadata}/android/release_notes.xml` file that our tooling generated (but will soon not have to anymore, see below)